PHP/README.md

```diff
@@ -205,7 +205,7 @@ On a Mac an alternative is to uncommented the line that says ```Include /private
 ### PHP version check has failed
 
 This is because the minimum proxy requirements are set to PHP version 5.4.2.  If this fails, the recommendation is to install the most recent stable PHP release.
-Other PHP versions may work, however PHP version 5.4.2 contains important security updates.
+A version of PHP greater than 5.4.2 should be used.
```

Comment:
i think it'd be preferable to just remove the old statement entirely and not replace it with anything.

Reply:
@jgravois - Yes, the `Occam's razor` method would work well here too :)